File: study/ts/script/ui/component/CommonButtonSwitchLevel0.ts
import CommonButtonSwitchLevel1 from "./CommonButtonSwitchLevel1";

const { ccclass, property } = cc._decorator;

@ccclass
export default class CommonButtonSwitchLevel0 extends CommonButtonSwitchLevel1 {

    constructor() {
        super();
        this._highLight = true;
        this._highLightImg = {
            normal: 'img_btn_ctrl_large02_nml',
            down: null,
            disable: null
        };
        this._normalImg = {
            normal: 'img_btn_ctrl_large01_nml',
            down: null,
            disable: null
        };
    }
}

File: study/lua/src/app/ui/component/CommonButtonSwitchLevel0.lua
local CommonButtonSwitchLevel1 = import(".CommonButtonSwitchLevel1")
local CommonButtonSwitchLevel0 = class("CommonButtonSwitchLevel0", CommonButtonSwitchLevel1)


function CommonButtonSwitchLevel0:ctor()
	CommonButtonSwitchLevel0.super.ctor(self)
    self._highLight = true
    self._highLightImg = {normal = "img_btn_ctrl_large01_nml",down = nil,disable =  nil}
    self._normalImg = {normal = "img_btn_ctrl_large02_nml",down = nil,disable =  nil}
end

return CommonButtonSwitchLevel0
